src/sass/partials/base/_base.scss

```diff
@@ -0,0 +1,312 @@
+    @include breakpoint(setting-get('max paragraph length')) {
+      max-width: 40em;
+    }
+  }
+
+  &--blockquote {
+    border-left: .25em solid setting-get('primary accent color');
+    color: color('gray', 80);
+    font-style: italic;
+    margin-left: 2em;
+    margin-right: 1em;
+    padding: 0 1.5em;
+  }
+
+  &--dt {
+    color: setting-get('secondary color');
+    font-weight: setting-get('bold font weight');
+  }
+
+  &--dd {
+    @include breakpoint(setting-get('max paragraph length')) {
+      max-width: 40em;
+    }
+  }
+
+  &--li {
+    margin-top: 0;
+
+    &::before {
+      content: '•';
+      margin-left: -1em;
+      padding-right: .8em;
+      text-align: right;
+    }
+
+    @include breakpoint(setting-get('max paragraph length')) {
+      max-width: 40em;
+    }
+  }
+
+  &--ul {
+    display: table;
+    list-style: none;
+    margin-left: 0;
+    padding-left: 0;
+  }
+
+  &--ol {
+    .base--li {
+      padding-left: .2em;
+
+      &::before {
+        display: none;
+      }
+    }
+  }
+
+  &--hr {
+    border: 0 none;
+    border-top: 2px solid color('grey', 20);
+    margin: setting-get('rhythm') / .25;
+    margin-left: 0;
+    margin-right: 0;
+    width: 15%;
+  }
+
+  &--table {
+    width: 100%;
+  }
+
+  &--caption {
+    color: color('gray', 50);
+    font-size: .8em;
+    padding: setting-get('rhythm') / 2;
+    padding-left: 0;
+    padding-right: 0;
+    padding-top: 0;
+    text-align: right;
+  }
+
+  &--thead {
+
+    color: setting-get('white');
+  }
+
+  &--tr {
+    &:nth-of-type(even) {
+      background-color: color('neutral-white', 30);
+
+      & .base--td::before {
+        background-color: color('teal', 60);
+        // responsive table zebra striping
+      }
+    }
+  }
+
+  &--th {
+    display: none;
+    font-size: .8em;
+    font-weight: setting-get('normal font weight');
+    padding: setting-get('rhythm');
+    text-align: left;
+    @media (min-width: 600px) {
+      background-color: setting-get('primary color');
+      display: table-cell;
+      font-size: 1em;
+    }
+  }
+
+  &--td {
+    display: flex;
+    font-size: .8em;
+    margin: 0;
+    padding: setting-get('rhythm') / 2;
+    @media (min-width: 600px) {
+      display: table-cell;
+      font-size: 1em;
+      padding: setting-get('rhythm');
+    }
+
+    &::before {
+      @include hyphens(auto);
+      background-color: setting-get('primary color');
+      color: setting-get('white');
+      content: attr(data-th)': ';
+      display: inline-block;
+      margin: -.5rem .5em -.5rem -.5rem;
+      min-width: 6.5em;
+      padding: setting-get('rhythm') / 2;
+      text-align: right;
+      width: 6.5em;
+      word-break: break-all;
+
+      @media (min-width: 600px) {
+        display: none;
+      }
+    }
+  }
+
+  &--kbd {
+    background-color: color('neutral-white', 40);
+    border: 1px solid color('gray', 10);
+    border-radius: 3px;
+    font-size: .8em;
+    padding: .2em .5em;
+  }
+
+  &--code {
```

Comment:
Take a look at the inline code styling that I've got in my [Flowtime theme](https://github.com/Snugug/snugug-flowtime/blob/master/partials/_base.scss#L125-L140). It's heavily based on GitHub's inline code styling (which I think is pretty much spot-on).

We also want to make sure that code is using a monospace font